token-lending/program/tests/helpers/mod.rs

```diff
@@ -587,7 +591,12 @@ impl TestLendingMarket {
             ],
             recent_blockhash,
         );
-        assert!(banks_client.process_transaction(transaction).await.is_ok());
+        let result = banks_client.process_transaction(transaction).await;
```

Comment:
By telling liquidate if it expects a success or not, we can return an actual result that can be used by the test to inspect validations later.